demo/src/app/components/+carousel/demos/custom-content/custom-content.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-carousel-custom-content',
+  styles: ['.jumbotron { margin: 0; background-color: #6c757d; color: #fff; }'],
```

Comment:
why do we need custom styles?
@tarusin

Reply:
in `.jumbotron` properties margin push down slider pagination, background is too light. that's why i decide make custom styles.
@valorkin